src/HistoricalSubscription.js

```diff
@@ -42,14 +42,8 @@ export default class HistoricalSubscription extends AbstractSubscription {
         return this.resendOptions
     }
 
-    finishResend(isNoResend = false) {
-        // if after a first resend request no messages are received (received ResendResponseNoResend),
-        // we wait for the response to the second resend request before considering the resend done (messages might have been stored in between)
-        if (isNoResend && !this.firstNoResendReceived) {
-            this.firstNoResendReceived = true
-        } else {
-            this._lastMessageHandlerPromise = null
-            this.emit('resend done')
-        }
+    finishResend() {
```

Comment:
I think this means `finishResend(true)` can be removed from https://github.com/streamr-dev/streamr-client-javascript/blob/93b49f90872186ac31246cc2a0cd46ec6a973c98/src/AbstractSubscription.js#L107